fetch_ivy_dependencies

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+
+import sys
+
+if sys.version_info[0] < 3 or sys.version_info[1] < 5:
+    print("This program requires Python version 3.5 or newer")
+    sys.exit(1)
+
+import os
+import argparse
+from lib import scalaplugin
+from lib import common
+from lib import logger
+
+def get_parser():
+    parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
+                                     description='Fetch Ivy Dependencies')
+    parser.add_argument('--scala', dest='scala', metavar='DIR', required=True,
+                        help='directory in which to install Scala',
+                        type=common.is_existing_dir)
+    parser.add_argument('--cache', dest='cache', metavar='DIR', required=True,
```

Comment:
Yeah I like --cache-dir` (and `--scala-dir` from the above comment). I'll change to use those. I'm trying to move away from using "ivy" because I think that was a mistake, but making it more clear these are directories makes sense.

Reply:
Oh whoops, I had updated my actual comment to say `--cache-dir` but forgot to update the suggested change, since I changed my mind in the middle of writing the comment.